tests/storage/restore.go

```diff
@@ -1564,15 +1565,24 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 					}
 				}
 
-				DescribeTable("should restore a vm that boots from a network cloned datavolumetemplate", func(restoreToNewVM bool) {
-					vm, vmi = createAndStartVM(tests.NewRandomVMWithDataVolumeCloneSourceAndUserData(
+				createVMFromSource := func() *v1.VirtualMachine {
+					dataVolume := libstorage.NewRandomDataVolumeWithPVCSource(
 						sourceDV.Namespace,
 						sourceDV.Name,
 						util.NamespaceTestDefault,
-						bashHelloScript,
-						snapshotStorageClass,
 						corev1.ReadWriteOnce,
-					))
+					)
+					libstorage.SetDataVolumePVCStorageClass(dataVolume, snapshotStorageClass)
+					libstorage.SetDataVolumePVCSize(dataVolume, "6Gi")
```

Comment:
does the pvc size needs to be this big?

Reply:
Probably not but it was being set to that size in the function I removed